docs/azure.md

```diff
@@ -0,0 +1,234 @@
+# Microsoft Azure Setup
+
+## Using Azure managed PostgreSQL
+To use Azure managed PostgreSQL with the `eoapi-k8s` chart, you need to set up the following:
+1. **Create an Azure PostgreSQL server**: You can create a PostgreSQL server using the Azure portal or the Azure CLI. Make sure to note down the server name, username, and password.
+2. **Create a PostgreSQL database**: After creating the server, create a database that will be used by the `eoapi-k8s` chart.
+3. **Configure firewall rules**: Ensure that the PostgreSQL server allows connections from your Kubernetes cluster's IP address. You can do this by adding a firewall rule in the Azure portal or using the Azure CLI.
```

Comment:
@geohacker I need your review on this part or a link to a reference documentation to setup an azure managed postgres with postgis and the related keyvault creation and access